src/components/TaxPicker/taxPickerPropTypes.js

```diff
@@ -8,14 +9,19 @@ const propTypes = {
     /** Callback to fire when a tax is pressed */
     onSubmit: PropTypes.func.isRequired,
 
-    /* Onyx Props */
     /** Collection of tax rates attached to a policy */
-    policyTaxRates: taxPropTypes,
+    taxRates: taxPropTypes,
+
+    /**
+     * Safe area insets required for reflecting the portion of the view,
+     * that is not covered by navigation bars, tab bars, toolbars, and other ancestor views.
+     */
+    insets: safeAreaInsetPropTypes.isRequired,
```

Comment:
Confirming you added this right? and is not added mistakenly in merge conflict?

Reply:
Yes, I added it.